README.md

```diff
@@ -124,15 +136,22 @@ common forms of colour blindness, and should also be discernable in
 greyscale. The standard palettes in this package have been designed to
 meet WCAG 2.0 accessibility guidance.
 
-Four of the created palettes (`main.palette`, `electric.brights`,
-`just.beachy` and `cycling.hills`) contain four colours each, and meet
-WCAG 2.0 AAA guidance. All colours have a contrast ratio of at least
+Two of the created accessible palettes (`travel.direction`, `country.lane`) 
+contain 8 colours each, and only one accessible palette (`joyful.journey`) 
+contains 6 colours each, and meet WCAG 2.0 AAA guidance. 
+
+Three of the created legacy palettes (`electric.brights`, `just.beachy`
```

Comment:
I would remove mentions of legacy palettes here, just because they're not as good